dusk/semantics.py

```diff
@@ -114,10 +114,10 @@ def get_field_dimension(field: sir.Field) -> LocationChain:
             == "unstructured_horizontal_dimension"
         )
         dimension = field.field_dimensions.unstructured_horizontal_dimension
-        if 0 < len(dimension.sparse_part):
-            return list(dimension.sparse_part)
+        if len(dimension.iter_space.chain) > 1:
+            return list(dimension.iter_space.chain)
```

Comment:
```suggestion
        return list(dimension.iter_space.chain)
```
+remove if/else entirely (github suggestions doesn't allow me to do the whole suggestion together...)